src/components/pipelines-metrics/PipelinesAverageDuration.tsx

```diff
@@ -100,11 +100,11 @@ const PipelinesAverageDuration: React.FC<PipelinesAverageDurationProps> = ({
               }
               scale={{ x: 'time', y: 'linear' }}
               domain={domainValue}
-              domainPadding={{ x: [30, 25], y: [30, 25] }}
+              domainPadding={{ x: [30, 25]}}
               height={150}
               width={400}
               padding={{
-                top: 20,
+                top: 40,
```

Comment:
It adds more space between the title and chart and it doesn't look good and reduces the height as well.